src/ApiCallable.php

```diff
@@ -134,63 +134,39 @@ private static function setLongRunnning($callable, $longRunningDescriptor)
         return $inner;
     }
 
-    private static function createUnaryApiCall($callable)
-    {
-        return function () use ($callable) {
-            list($response, $status) =
-                call_user_func_array($callable, func_get_args())->wait();
-            if ($status->code == Grpc\STATUS_OK) {
-                return $response;
-            } else {
-                throw ApiException::createFromStdClass($status);
-            }
-        };
-    }
-
-    private static function createGrpcStreamingApiCall($callable, $grpcStreamingDescriptor)
```

Comment:
@michaelbausor How do you see us replacing this logic? I see two options:

1. Rename "grpcStreamingDescriptor" to "streamingDescriptor" and pass this to both Rest and GRPC calls as an option using `ApiCallable`
1. Remove "grpcStreamingDescriptor" all together and have the GAPIC client return the proper call object from the transport layer

Reply:
I like 2. The return types on the transport interface can be `BidiStream` (or `BidiStreamInterface`), then the details of constructing the `BidiStream` object can be handled in the grpc transport implementation (same for Server and Client streams).